crates/release_plz/tests/all/helpers/test_context.rs

```diff
@@ -81,10 +81,14 @@ impl TestContext {
         context
     }
 
-    pub async fn merge_release_pr(&self) {
+    pub async fn merge_release_prs(&self) {
         let opened_prs = self.opened_release_prs().await;
         assert_eq!(opened_prs.len(), 1);
-        self.gitea.merge_pr_retrying(opened_prs[0].number).await;
+        self.merge_release_pr(&opened_prs[0]).await;
```

Comment:
this function is called "merge_release_prs" but you only merge the first one 🤔

Reply:
Good catch. This is intentional, and I wanted to discuss it but forgot to point it out. 

This is the original `merge_release_pr` function. I was hesitant to change its behavior since I was unsure whether it would impact the existing tests that relied on this specific implementation.

The tests do pass if we actually close all PRs, so it does not seem to be a problem. It might change some subtle behavior I am unaware of though.